### PR TITLE
support internvl2-1b

### PR DESCRIPTION
**Pull request by RunningLeon**
## Motivation

Support [OpenGVLab/InternVL2-1B](https://huggingface.co/OpenGVLab/InternVL2-1B) for pytorch engine

related issue: https://github.com/InternLM/lmdeploy/issues/1963

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: lmdeploy/turbomind/supported_models.py [hunk 1]
```diff
@@ -62,6 +62,12 @@ def is_supported(model_path: str):
     """  # noqa: E501
     import os
 
+    def _is_head_dim_ok(cfg):
```
Comment (lvhan028) [inline lmdeploy/turbomind/supported_models.py:65]: Let's change the function signature to `_is_head_dim_128`

**Review by zhyncs — COMMENTED** [review]
@RunningLeon I used the method mentioned here https://github.com/InternLM/lmdeploy/issues/1959#issuecomment-2216656187

And the InternVL2 1B's result is not same as InternVL2 2B
```
{"id":"1","object":"chat.completion","created":1720606716,"model":"InternVL2-1B","choices":[{"index":0,"message":{"role":"assistant","content":"true","tool_calls":null},"logprobs":null,"finish_reason":"stop"}],"usage":{"prompt_tokens":1952,"total_tokens":1953,"completion_tokens":1}}
```

**Comment by zhyncs** [comment]
transformers result for InterVL2 1B is

 Assistant: ```json
{
  "felled_tree": false,
  "slidanele": false,
  "flooded_shop": False
}
\`\`\`

**Comment by zhyncs** [comment]
> not same as

The issue I encountered was caused by multiple local environments. Just ignore it.

**Review by zhyncs — APPROVED** [review]

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**